services/bls_aggregation/blsagg.go

```diff
@@ -314,6 +326,43 @@ func (a *BlsAggregatorService) singleTaskAggregatorGoroutineFunc(
 
 }
 
+type Wrapper struct {
```

Comment:
can we make this private? `type wrapper struct {...}`

Reply:
refined